### PR TITLE
feat(hatcherySwarm): force pull if tag is latest

### PR DESCRIPTION
**Pull request by yesnault**

### CODE REVIEW

File: engine/hatchery/swarm/swarm.go [hunk 1]
```diff
@@ -355,7 +355,7 @@ checkImage:
 		}
 	}
 
-	if !imageFound {
+	if !imageFound || strings.HasSuffix(model.Image, ":latest") {
```
Comment (fsamin) [inline engine/hatchery/swarm/swarm.go]: humm, maybe we could also avoid do the ListImages()  above

Comment (yesnault) [inline engine/hatchery/swarm/swarm.go]: yep, fixed

Comment (yesnault) [inline engine/hatchery/swarm/swarm.go]: in next commit...

Comment (yesnault) [inline engine/hatchery/swarm/swarm.go]: fixed

**Pull request merged**